packages/terra-graphs/src/components/Timeline/Timeline.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint import/no-unresolved: off */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+// import './Timeline.module.scss';
```

Comment:
Don't we need this scss?

Reply:
removed here: a5aed21a6d74dc81b9b61192b5dddcadc455a452